extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/VertxHttpRecorder.java

```diff
@@ -532,9 +532,11 @@ private static void doServerStart(Vertx vertx, HttpBuildTimeConfig httpBuildTime
         }
 
         int eventLoopCount = eventLoops.get();
-        int ioThreads;
+        final int ioThreads;
         if (httpConfiguration.ioThreads.isPresent()) {
             ioThreads = Math.min(httpConfiguration.ioThreads.getAsInt(), eventLoopCount);
+        } else if (launchMode.isDevOrTest()) {
```

Comment:
Are you sure you want to do this in dev-mode as well?

Reply:
yes, why not?

Reply:
Because it's pretty significant difference between dev mode and prod mode

Reply:
I realize I might be missing something, but I don't expect people to perform load tests in dev mode.

On the other hand, it keeps resource usage under control when running interactive exploration sessions, possibly having multiple Quarkus based microservices running. 

Other than performance aspects, I don't think it would have semantic changes that impact developer's ability to explore things? 

( other than facilitating debugging by not having breakpoints hit by 100 threads )

But yeah sure I get that it's debatable, no rush. That's also why I pinged Stuart too, so at least he's aware.

We can totally start doing it for tests only.

Reply:
Yeah, I understand the rationale completely. It's always a delicate balance... 

I am not totally against doing this in dev-mode, I guess I am +0 :)